aerospike/datadog_checks/aerospike/data/conf.yaml.example

```diff
@@ -67,13 +67,14 @@ instances:
 
 init_config:
 
+  ## @param mappings - list of strings - required
   ## Rate mappings. DO NOT EDIT unless you know what you're doing.
   ## See https://github.com/aerospike/aerospike-collectd/blob/develop/aerospike_schema.yaml
   ## for the definitive list
-
+  #
   mappings:
-    - basic_scans_failed    # moved to NS scan_basic_error
```

Comment:
why deleting the comments? Do we have the equivalents in the changelog?

Reply:
Alright bringing back comments and i'm going to make them match the new logic then